x-pack/test/api_integration/apis/security_solution/authentications.ts

```diff
@@ -44,9 +44,10 @@ export default function ({ getService }: FtrProviderContext) {
             fakePossibleCount: 3,
             querySize: 1,
           },
-          defaultIndex: ['auditbeat-*', 'filebeat-*', 'packetbeat-*', 'winlogbeat-*'],
+          defaultIndex: ['auditbeat-*'],
           docValueFields: [],
           inspect: false,
+          wait_for_completion_timeout: '10s',
```

Comment:
Just curious how we settled on the `10s` timeout?

Reply:
Sometimes the request just takes longer than expected to return. I want to put the timeout long enough to wait until the result comes back so I am able to verify the result.

Reply:
I checked the test here: https://github.com/elastic/kibana/blob/master/x-pack/test/api_integration/apis/search/search.ts#L18

They use a very long timeout to avoid returning just partial data, so I do the same here.